apps/docs/src/app/platform/platform-documentation.module.ts

```diff
@@ -144,7 +151,13 @@ import { ScrollingModule } from '@angular/cdk/scrolling';
         PlatformMenuButtonExampleComponent,
         PlatformMenuButtonCozyExampleComponent,
         PlatformMenuButtonCompactExampleComponent,
-        PlatformMenuButtonHeaderComponent
+        PlatformMenuButtonHeaderComponent,
+        PlatformInfoLabelDocsComponent,
+        PlatformInfoLabelHeaderComponent,
+        PlatformInfoLabelExampleComponent,
+        PlatformInfoLableNumericIconExampleComponent,
+        PlatformInfoLableTextExampleComponent,
+        PlatformInfoLableTextIconExampleComponent,
     ],
     imports: [
         FundamentalNgxCoreModule,
```

Comment:
HI @fkolar all the changes are in place, since the i have squashed the changes you are unable to see those changes, PFA screen shots.

<img width="980" alt="Screenshot 2020-05-18 at 9 04 54 AM" src="https://user-images.githubusercontent.com/53534379/82172049-b97b7f80-98e6-11ea-98fc-bb211e9e6e31.png">
![Uploading Screenshot 2020-05-18 at 9.03.35 AM.png…]()